PartsApp/AddEmployeeForm.cs

```diff
@@ -415,9 +414,10 @@ private void FillTheForm(Employee employee)
                                                            //photoPictureBox.Image = (employee.Photo != null) ? new Bitmap(Image.FromFile(employee.Photo), photoPictureBox.Size) : null;
             if (employee.Photo != null)
             {
-                if (System.IO.File.Exists(System.IO.Path.GetFullPath(employee.Photo)))
+                string fullPhotoPath = $"{FilesStorageHelper.DataDirectoryPath}\\{employee.Photo}";
```

Comment:
Double-check that this works properly.
It looks to me like 'Employee' folder is missing here.

Reply:
it's ok, because Employee (as well as SparePart) Photo property contains directory's title 
![image](https://github.com/user-attachments/assets/147468d5-27c3-4d81-bd8d-25a6a0de2a13)

![image](https://github.com/user-attachments/assets/63a850aa-f0e7-4580-a738-2e12fcdd530a)